### DISCUSSION TITLE
Is it possible to hide these dots when using multiple columns?

### DISCUSSION THREAD

**Discussion by isak102** [davatorium/rofi#1647] [General]
I am using multiple columns in a rofi script and I want to hide these 3 dots at the end of every line, is there an option for this? Couldn't find anything in the documentation 

![24-06-2022_01-11](https://user-images.githubusercontent.com/90389894/175501742-f6316137-a6c5-4fe5-826b-6a95c134c695.png)

**Comment by DaveDavenport** [comment]
it indicates part of the text is cut off, make the text entries shorter.

**Reply by isak102** [reply]
Yeah, I realised almost immidiately after I posted that there was a bunch of whitespace after each entry
